src/App.vue

```diff
@@ -3,6 +3,15 @@
     <navigation-drawer />
     <v-main>
       <router-view />
+
+      <!-- temporary locale changer -->
+      <div class="locale-changer">
+        <select v-model="$i18n.locale">
+          <option v-for="(lang, i) in langs" :key="`Lang${i}`" :value="lang">
+            {{ lang }}
+          </option>
+        </select>
+      </div>
```

Comment:
This will let you switch between active locales. We'd need to move the locales in to some sort of config, or better yet just use the keys of valid translations once they're completed. Something like:
```
Object.keys(messages);
```

Reply:
Moved to using the global `$i18n.availableLanguages`.